sdk/core/core-arm/package.json

```diff
@@ -87,7 +87,11 @@
     "test": "npm run build && npm run unit-test && npm run integration-test",
     "unit-test": "npm run unit-test:node && npm run unit-test:browser",
     "unit-test:browser": "echo skipped",
-    "unit-test:node": "nyc mocha --require ts-node/register --timeout 50000 --reporter ../../../common/tools/mocha-multi-reporter.js --colors \"test/**/*.ts\"",
+    "unit-test:node": "mocha --require ts-node/register --timeout 50000 --reporter ../../../common/tools/mocha-multi-reporter.js --colors \"test/**/*.ts\"",
+    "cover:unit:browser": "nyc npm run unit-test:browser",
```

Comment:
`nyc` in the command is meant for mocha and node tests, not for browser.
`cover:unit:browser` can remain same as `unit-test:browser`.

Reply:
why not?

Reply:
Browser test coverage works differently, we have `karma.conf.js` with karma-coverage and karma-remap-istanbul reporters with configs to generate the code coverage.

Browser test command is usually `karma start --single-run`, I don't think `nyc karma` works the way `nyc mocha` does and that's what I meant in the previous comment.

Reply:
ah, I see. Ok, will remove it.